pytensor/link/utils.py

```diff
@@ -704,6 +704,8 @@ def fgraph_to_python(
     get_name_for_object
         A function used to provide names for the objects referenced within the
         generated function.
+    unique_name
```

Comment:
Don't pass through kwargs here but explicitly with the default being what we had before